dali/test/python/test_operator_reshape.py

```diff
@@ -289,3 +292,35 @@ def _test_reinterpret_wildcard_shape(device):
 def test_reinterpret_wildcard_shape():
     for device in ["cpu", "gpu"]:
         yield _test_reinterpret_wildcard_shape, device
+
+def get_data(shapes):
+    return [np.empty(shape, dtype = np.uint8) for shape in shapes]
+
+@pipeline_def
+def reshape_pipe(shapes, src_dims=None, rel_shape=None):
+    source_fn = partial(get_data, shapes)
+    data = fn.external_source(lambda: get_data(shapes), batch=True, device = "cpu")
+    out = fn.reshape(data, src_dims=src_dims, rel_shape=rel_shape)
+    return out
+
+def _testimpl_reshape_src_dims_arg(src_dims, rel_shape, shapes, expected_out_shapes):
```

Comment:
Please add separate a test function covering errors (wrong arguments, removing dimensions with extent > 1, different length of src_dims and rel_shape args, etc...)